dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/ConfigTestsWeb.war/src/com/ibm/ws/ejbcontainer/bindings/configtests/web/BindToServerRootServlet.java

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************n * Copyright (c) 2020 IBM Corporation and others.n * All rights reserved. This program and the accompanying materialsn * are made available under the terms of the Eclipse Public License v1.0n * which accompanies this distribution, and is available atn * http://www.eclipse.org/legal/epl-v10.htmln *n * Contributors:n *     IBM Corporation - initial API and implementationn *******************************************************************************/
```

Comment:
Copyright is missing returns

Reply:
I forgot my eclipse does this for new files oops, I'll fix it for the next PR